ci_framework/roles/libvirt_manager/templates/ci-network-data.yml.j2

```diff
@@ -0,0 +1,50 @@
+{#
+The ci_job_networking is exposed by the reproducer role. It namely exposes
+the CI job configuration passed down to zuul so that we can ensure we're
+using the same networks.
+#}
+{% set ns = namespace(
+    ip4='',
+    gw4='',
+    iface=cifmw_reproducer_private_nic,
+    config_nm=false, ip_suffix=0)
+-%}
+{% if vm_type == 'controller' -%}
+{%   set ns.gw4 = cifmw_reproducer_ctl_gw4 -%}
+{%   set ns.ip4 =  ci_job_networking['instances']['controller']['networks']['default']['ip'] %}
+{% elif vm_type == 'crc' -%}
+{%   set ns.ip_suffix = 10 -%}
+{%   set ns.iface = cifmw_reproducer_crc_private_nic -%}
+{%   set ns.gw4 = cifmw_reproducer_crc_gw4 -%}
+{%   set ns.config_nm = true -%}
+{%   set ns.ip4 =  ci_job_networking['instances']['crc']['networks']['default']['ip'] %}
+{% else -%}
+{%   set ns.ip_suffix = 100+vm_id -%}
+{%   set ns.ip4 = ci_job_networking['instances']['compute-' ~ vm_id]['networks']['default']['ip']%}
+{%   set ns.gw4 = cifmw_reproducer_ctl_gw4 -%}
+{% endif -%}
+{% set default_mtu = ci_job_networking.networks['default'].mtu | default('1500') -%}
```

Comment:
Can you move all `set` unrelated to VM config at the top of the file?

Reply:
done